src/openseadragon.js

```diff
@@ -249,6 +249,11 @@
   *     image though it is less effective visually if the HTML5 Canvas is not
   *     availble on the viewing device.
   *
+  * @property {Number} [smoothTileEdgesMinZoom=1.1]
+  *     A zoom percentage ( expressed as a number between 0 and 1 ) of the highest
+  *     resolution level. When zoomed in beyond this value alternative compositing will
+  *     be used to smooth out the edges between tiles. This WILL have a performance impact.
```

Comment:
I don't know that the performance impact is great enough to warrant caps on "will"...we don't need to scare people away.